NewBank/newbank/server/NewBank.java

```diff
@@ -52,4 +57,40 @@ private String showMyAccounts(CustomerID customer) {
 		return (customers.get(customer.getKey())).accountsToString();
 	}
 
+	private String move(CustomerID customer, String[] splitRequest){
+		// check there are 4 things in the request
+		if (splitRequest.length != 4){
+			return "FAIL";
+		}
+		// check both bank accounts exist for the customer
+		ArrayList<Account> customerAccounts = customers.get(customer.getKey()).getAccountList();
+		double amount = Double.parseDouble(splitRequest[1]);
```

Comment:
MOVE works well for all the inputs I tested, except when the amount input is not a double or integer (i.e. MOVE a Main Savings), and it infinitely prints null. For this, my suggestion would be a try-catch block for a NumberFormatException.

Reply:
Great, thanks!

Reply:
<img width="412" alt="Screen Shot 2021-03-15 at 09 31 11" src="https://user-images.githubusercontent.com/80157394/111132399-37450180-8571-11eb-8b38-0bab76057ccc.png">

Done :)